### DISCUSSION TITLE
Will the nunif browser plugin be created?

### DISCUSSION THREAD

**Discussion by My12123** [nagadomi/nunif#28] [Ideas]
Will an extension be created for Google Chrome? Available at https://chrome.google.com/webstore/category/extensions

**Comment by nagadomi** [comment]
same as https://github.com/nagadomi/waifu2x/discussions/446
I do not plan to develop a chrome-extension at this time.

**Comment by elecimage** [comment]
i have 2 gpus.
Haw can i use all gpu  in iw3?

**Reply by nagadomi** [reply]
Currently it is not supported.
For video processing, it can be supported.
For image processing, only one GPU can be used because batch-size=1.

**Reply by elecimage** [reply]
Ok thank you i want to video support~
Cheers!

**Reply by nagadomi** [reply]
@elecimage  It seems that it is not easy to implement due to the class structure of ZoeDepth.
I created the issue https://github.com/nagadomi/nunif/issues/59
